examples/v1beta1/sdk/cmaes-and-resume-policies.ipynb

```diff
@@ -74,11 +98,15 @@
   {
    "cell_type": "code",
    "execution_count": 46,
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
    "outputs": [],
    "source": [
     "# Experiment name and namespace.\n",
-    "namespace = \"kubeflow-user-example-com\"\n",
+    "namespace = \"kubeflow\"\n",
     "experiment_name = \"cmaes-example\"\n",
     "\n",
     "metadata = V1ObjectMeta(\n",
```

Comment:
Can we add `parameters` tag in `metadata` and allow args in papermill rewrite them like: https://github.com/kubeflow/training-operator/pull/2274?